// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/moduleDeclarationExportStarShadowingGlobalIsNameable.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,9 +1,9 @@
 
 //// [index.d.ts.map]
-{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,CAAC,MAAM,CAAC;IACX,UAAU,OAAO;QACb,QAAQ,EAAE,MAAM,CAAC;KACpB;IACD,UAAU,GAAG;QACT,QAAQ,EAAE,MAAM,CAAC;KACpB;CACJ;AACD,OAAO,KAAK,KAAK,MAAM,SAAS,CAAC;AACjC,eAAO,MAAM,IAAI,YAAa,aAAa,QAAQ,SAAS,KAAG,IAAU,CAAC"}
+{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,OAAO,CAAC,MAAM,CAAC;IACX,UAAU,OAAO;QACb,QAAQ,EAAE,MAAM,CAAC;KACpB;IACD,UAAU,GAAG;QACT,QAAQ,EAAE,MAAM,CAAC;KACpB;CACJ;AACD,OAAO,KAAK,KAAK,MAAM,SAAS,CAAC;AACjC,eAAO,MAAM,IAAI,GAAI,OAAO,EAAE,KAAK,CAAC,OAAO,EAAE,IAAI,EAAE,KAAK,CAAC,GAAG,KAAG,IAAU,CAAC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBnbG9iYWwgew0KICAgIGludGVyZmFjZSBBY2NvdW50IHsNCiAgICAgICAgc29tZVByb3A6IG51bWJlcjsNCiAgICB9DQogICAgaW50ZXJmYWNlIEFjYyB7DQogICAgICAgIHNvbWVQcm9wOiBudW1iZXI7DQogICAgfQ0KfQ0KaW1wb3J0ICogYXMgbW9kZWwgZnJvbSAiLi9tb2RlbCI7DQpleHBvcnQgZGVjbGFyZSBjb25zdCBmdW5jOiAoYWNjb3VudDogbW9kZWwuQWNjb3VudCwgYWNjMjogbW9kZWwuQWNjKSA9PiB2b2lkOw0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9aW5kZXguZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sQ0FBQyxNQUFNLENBQUM7SUFDWCxVQUFVLE9BQU87UUFDYixRQUFRLEVBQUUsTUFBTSxDQUFDO0tBQ3BCO0lBQ0QsVUFBVSxHQUFHO1FBQ1QsUUFBUSxFQUFFLE1BQU0sQ0FBQztLQUNwQjtDQUNKO0FBQ0QsT0FBTyxLQUFLLEtBQUssTUFBTSxTQUFTLENBQUM7QUFDakMsZUFBTyxNQUFNLElBQUksWUFBYSxhQUFhLFFBQVEsU0FBUyxLQUFHLElBQVUsQ0FBQyJ9,ZXhwb3J0ICogZnJvbSAiLi9hY2NvdW50IjsK
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBnbG9iYWwgew0KICAgIGludGVyZmFjZSBBY2NvdW50IHsNCiAgICAgICAgc29tZVByb3A6IG51bWJlcjsNCiAgICB9DQogICAgaW50ZXJmYWNlIEFjYyB7DQogICAgICAgIHNvbWVQcm9wOiBudW1iZXI7DQogICAgfQ0KfQ0KaW1wb3J0ICogYXMgbW9kZWwgZnJvbSAiLi9tb2RlbCI7DQpleHBvcnQgZGVjbGFyZSBjb25zdCBmdW5jOiAoYWNjb3VudDogbW9kZWwuQWNjb3VudCwgYWNjMjogbW9kZWwuQWNjKSA9PiB2b2lkOw0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9aW5kZXguZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImluZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLE9BQU8sQ0FBQyxNQUFNLENBQUM7SUFDWCxVQUFVLE9BQU87UUFDYixRQUFRLEVBQUUsTUFBTSxDQUFDO0tBQ3BCO0lBQ0QsVUFBVSxHQUFHO1FBQ1QsUUFBUSxFQUFFLE1BQU0sQ0FBQztLQUNwQjtDQUNKO0FBQ0QsT0FBTyxLQUFLLEtBQUssTUFBTSxTQUFTLENBQUM7QUFDakMsZUFBTyxNQUFNLElBQUksR0FBSSxPQUFPLEVBQUUsS0FBSyxDQUFDLE9BQU8sRUFBRSxJQUFJLEVBQUUsS0FBSyxDQUFDLEdBQUcsS0FBRyxJQUFVLENBQUMifQ==,ZXhwb3J0ICogZnJvbSAiLi9hY2NvdW50IjsK
 
 
 //// [model/index.d.ts.map]
 {"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":"AAAA,cAAc,WAAW,CAAC"}
